.github/workflows/main.yml

```
@@ -24,10 +24,12 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: poetry
       - name: Install dependencies
-        run: poetry install --no-interaction --no-root --with dev
+        run: poetry install --no-interaction --no-root --with dev --with examples
```

`--with` supports comma separated syntax too 🙃
```suggestion
        run: poetry install --no-interaction --no-root --with dev,examples
```